site/src/styles/global.css

```
@@ -175,4 +182,21 @@
+  /* PP Formula sits high on its baseline; nudge it down to optically center
+     within fixed-height buttons/badges. Not horizontal centering. */
+  .ppformula-optical-center {
+    display: inline-block;
+    position: relative;
+    top: 0.19em;
+  }
+
+  /* Compact variant for small controls (tabs, pill buttons at ~text-xs) where
+     the label sits beside an icon. A lighter em-relative nudge so the label
+     optically centers against the icon without dropping below it. Scales with
+     font-size, so it holds across breakpoints.*/
```

Stylelint (17.13.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)